src/governance/timelock/utils/call_impls.cairo

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.15.0-rc.0 (governance/timelock/utils/call_impls.cairo)
+
+use core::hash::{HashStateTrait, HashStateExTrait, Hash};
+use starknet::ContractAddress;
+use starknet::account::Call;
+
+pub(crate) impl HashCallImpl<S, +HashStateTrait<S>, +Drop<S>> of Hash<@Call, S> {
```

Comment:
Why `Hash<@Call, S>` instead of `Hash<Call, S>`

Reply:
Not necessary now. It was for serde initially